release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -456,9 +456,36 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-11-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-11-oc-commands-flags-tokens-deprecated"]
+==== OpenShift CLI (oc) commands and flags for requesting tokens deprecated
+
+The following `oc` commands and flags for requesting tokens are now deprecated:
+
+* The `oc serviceaccounts create-kubeconfig` command
+* The `oc serviceaccounts get-token` command
+* The `oc serviceaccounts new-token` command
+* The `--service-account/-z` flag for the `oc registry login` command
+
+Users should use the `oc create token` command instead to request tokens.
+
 [id="ocp-4-11-removed-features"]
 === Removed features
 
+[id="ocp-4-11-oc-rhel-7"]
+==== RHEL 7 support for the OpenShift CLI (oc) removed
+
+Support for using Red Hat Enterprise Linux (RHEL) 7 with the OpenShift CLI (`oc`) has been removed. If you use the OpenShift CLI (`oc`) with RHEL, you must use RHEL 8.
```

Comment:
You could reverse the components in "Support for using Red Hat Enterprise Linux (RHEL) 7 with the OpenShift CLI (`oc`) has been removed" to prevent readers from potentially interpreting this to mean that RHEL 7 support is removed if they use `oc`. For example:

> Support for using the OpenShift CLI (`oc`) on Red Hat Enterprise Linux (RHEL) 7 has been removed.